src/helpers/signupSchema.js

```diff
@@ -0,0 +1,51 @@
+import Joi from '@hapi/joi';
+
+export default {
+  signup: (userData) => {
+    const schema = {
+      firstName: Joi.string()
+        .trim()
+        .required()
```

Comment:
Good initiative creating this `SignupSchema` validation file. But I suggest you create another file whereby all validations that are clients of reusability like email and password validation be added there then its children can reuse those validations.